logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/asset/SendAssetMessageUseCase.kt

```diff
@@ -124,7 +127,12 @@ internal class SendAssetMessageUseCaseImpl(
             status = Message.Status.PENDING,
             editStatus = Message.EditStatus.NotEdited
         )
-        messageRepository.persistMessage(message).map { message }
+        messageRepository.persistMessage(message)
+            .map { message }
+            .onSuccess {
+                conversationRepository.updateConversationNotificationDate(message.conversationId, message.date)
```

Comment:
maybe we can have update date function in the conversationRepository and call these 2 functions in it ? 
So we won't forget to call one and not the other in the future cases !

Reply:
i think this problem is solved by new changes that Mohamad suggested :)